src/app/clusters/icd-management-server/ICDManagementCluster.cpp

```diff
@@ -316,8 +316,11 @@ Status ICDManagementServer::RegisterClient(CommandHandler * commandObj, const Co
     {
         // Notify subscribers that the first entry for the fabric was successfully added
         TriggerICDMTableUpdatedEvent();
+        emberAfAttributeChanged(/* endpoint = */ 0, IcdManagement::Id, IcdManagement::Attributes::OperatingMode::Id,
```

Comment:
This is not the right function.  The right function is `MatterReportingAttributeChangeCallback`, no?

Reply:
And please coordinate with https://github.com/project-chip/connectedhomeip/pull/41499, after which the right way to mark dirty will change again